webpack/assets/javascripts/react_app/components/FactCharts/FactChartHelpers.js

```diff
@@ -0,0 +1,17 @@
+import {
+  OPEN_FACT_CHART_MODAL,
+  CLOSE_FACT_CHART_MODAL,
+} from './FactChartConstants';
+
+export const showModal = (id, title) => {
```

Comment:
why having actions in a helper file? 
please notice that the `showModal` action also exists in the `actions` file

Reply:
Thanks, as no other file uses this functions the Helpers file is now deleted.